core/lib/progressbar.ts

```diff
@@ -0,0 +1,191 @@
+import {clamp} from './services/checks';
+import type {ConfigValidator, PropsConfig} from './services';
+import {stateStores, typeBoolean, typeNumber, typeString, writablesForProps} from './services';
+import type {SlotContent, Widget, WidgetSlotContext} from './types';
+import {computed} from '@amadeus-it-group/tansu';
+
+export type ProgressbarContext = WidgetSlotContext<ProgressbarWidget>;
+
+export interface ProgressbarCommonPropsAndState {
+	/**
+	 * the minimum value
+	 * @defaultValue 0
+	 */
+	minimum: number;
+
+	/**
+	 * the maximum value
+	 * @defaultValue 100
+	 */
+	maximum: number;
```

Comment:
Maybe max / min would be a more common name (as Math.min / Math.max for example).

(something to do in the rating, as it does not follow this convention)